src/UI/Component/DropDown/DropDownItem.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Copyright (c) 2017 Alexander Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\DropDown;
+
+use ILIAS\UI\Component\Clickable;
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Hoverable;
+use ILIAS\UI\Component\JavaScriptBindable;
+
+/**
+ * Drop down items
+ */
+interface DropDownItem extends Component, JavaScriptBindable, Clickable, Hoverable {
```

Comment:
Oha, Dividers. Just used your idea in #550, so in the meantime I would support it and also have some kind of implementation ready. I think the handling sections would be more complicated (for the appointment lists you refused some "handling of grouping there", if I remember correctly). From the UI point of view we deal with a sequence of links and dividers, so I have a preference to implement your initial proposal. :-)

I can implement Links and Dividers, if we agree on where to place it in the hierarchy. Both top level?